### DISCUSSION TITLE
能否支持免登陆？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [gitalk/gitalk#475] [General]
因为有些人可能没有 GitHub 账号，希望能支持免 GitHub 登陆。

**Comment by gavinkou** [comment]
同问

**Comment by ghost** [comment]
不登陆没法发issue

**Reply by GanZhiXiong** [reply]
实现不登录查看评论应该实现吧？

**Reply by ghost** [reply]
不是本来就可以吗？

**Reply by ghost** [reply]
![屏幕截图 2022-01-16 114730](https://user-images.githubusercontent.com/37929478/149646498-3ef9c418-34ff-4206-ba85-585483352721.png)

**Reply by GanZhiXiong** [reply]
抱歉，我记得以前好像是不可以的，那可能我记错了
